inference-engine/tests/unit/inference_engine/ie_blob_test.cpp

```diff
@@ -393,5 +399,92 @@ TEST_F(BlobTests, makeRoiBlobWrongSize) {
 
     // try to create ROI blob with wrong size
     InferenceEngine::ROI roi = {0, 1, 1, 4, 4};  // cropped picture with: id = 0, (x,y) = (1,1), sizeX (W) = 4, sizeY (H) = 4
-    ASSERT_THROW(make_shared_blob(blob, roi), InferenceEngine::details::InferenceEngineException);
+    ASSERT_THROW(blob->CreateROIBlob(roi), InferenceEngine::details::InferenceEngineException);
+}
+
+namespace InferenceEngine {
+    bool operator==(const ROI&l, const ROI&r) {
```

Comment:
Nitpick: spaces between `ROI&` and `l`? There are typical names for such operators in C++: `lhs` and `rhs`..

Reply:
Do you consider to provide an implementation for that method outside of tests in library?

Reply:
I expected spaces to be caught by the style checker.
Moving out - not really, comparing the structs is for tests only

Reply:
I mean, such an operator has a great chance to be useful outside of tests scope, so moving it out (that's cheap operation, right?) rather useful..

Reply:
That might be reasonable

Reply:
moved